src/App.css

```diff
@@ -102,9 +92,16 @@ main > h1 + * {
   margin-top: 1rem;
 }
 
-.map__inner {
-  width: 100%;
+.view__content {
```

Comment:
Thank you for fixing our navigation. I just checked it, and it works! Congrats, great job Katia 😄! Now, we can add many items, and our navbar still will be visible.

![image](https://user-images.githubusercontent.com/8689897/126716531-a4a85ca7-2fc6-48d4-83b1-3ed5fe322480.png)